tests/wicked/advanced/sut/t15_macvtap_legacy.pm

```diff
@@ -27,28 +27,19 @@ sub run {
     my $config     = '/etc/sysconfig/network/ifcfg-macvtap1';
     my $ip_address = $self->get_ip(type => 'macvtap', netmask => 1);
     $ip_address =~ s'/'\\/';
-    my $ref_ip = $self->get_ip(type => 'host', netmask => 0, is_wicked_ref => 1);
     $self->get_from_data('wicked/ifcfg/macvtap1',    $config);
     $self->get_from_data('wicked/ifcfg/macvtap_eth', '/etc/sysconfig/network/ifcfg-' . $ctx->iface());
-    $self->get_from_data('wicked/check_macvtap.c',   'check_macvtap.c', executable => 1);
-    zypper_call('in gcc');
-    assert_script_run('gcc ./check_macvtap.c -o check_macvtap');
-    script_run('chmod +x ./check_macvtap');
-    file_content_replace($config, iface => $ctx->iface(), ip_address => $ip_address);
```

Comment:
you can then remove file_content_replace from "use" section and just do:

    use utils 'systemctl';